runtime/queries/fsharp/highlights.scm

```diff
@@ -0,0 +1,33 @@
+"module" @keyword.other
```

Comment:
We use these scopes for syntax highlights queries: https://docs.helix-editor.com/master/themes.html#syntax-highlighting

Some of these like keyword.other and keyword.type should use those scopes instead